migrations/postgres/20250426151069_did_secret_keys_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE dataset_did_secret_keys(
```

Comment:
All our migrations have separating comments. Please follow the format of other migrations.